source/features/bugs-tab.tsx

```diff
@@ -11,6 +11,24 @@ import {getRepo} from '../github-helpers';
 import SearchQuery from '../github-helpers/search-query';
 import abbreviateNumber from '../helpers/abbreviate-number';
 
+const cacheKey = (): string => __filebasename + ':' + getRepo()!.nameWithOwner;
+
+async function highlightBugsTabOnIssuePage(): Promise<void | false> {
+	if (!await cache.get<boolean>(cacheKey())) {
```

Comment:
Use `countBugs()`, it's a memoized promise so it'll work without race conditions.